core/app/services/workarea/add_multiple_cart_items/item.rb

```diff
@@ -23,7 +23,7 @@ def save
         return @item&.persisted? if defined?(@item)
 
         order.add_item(item_params).tap do |result|
-          @item = order.items.find_existing(sku, customizations&.to_h) if result
+          @item = order.items.find_existing(sku, item_params) if result
```

Comment:
Anyone know how many plugins use `find_existing`?

Reply:
https://stash.tools.weblinc.com/plugins/servlet/search?q=find_existing

Reply:
I found this one use in BOPUS: https://github.com/workarea-commerce/workarea-bopus/blob/64df917e175e3c584ad316a2a83ef3eeeee91103/app/models/workarea/order.decorator

And this one in Channel Advisor: https://stash.tools.weblinc.com/plugins/servlet/search?q=project%3AWP%20find_existing

Reply:
Ok, I guess not too bad.